### DISCUSSION TITLE
Font-awesome extension not recognized on my machine

### DISCUSSION THREAD

**Discussion by Shivansh-Mittal** [squidfunk/mkdocs-material#3974] [Q&A] [answered]
INFO     -  Building documentation...
ERROR    -  Config value: 'markdown_extensions'. Error: Failed loading extension
            "fontawesome_markdown".
Aborted with 1 Configuration Errors!

![image](https://user-images.githubusercontent.com/81565355/171106794-10ce946b-5533-4541-8651-496f43ba0a72.png)

It is showing this message in my VS code terminal.
Help me to resolve this.

**Comment by squidfunk** [comment, marked as answer]
If you want to use `fontawesome_markdown`, you need to install it first, but there's no need, as Material for MkDocs integrates with FontAwesome natively. The syntax is just a little different. Enter `fontawesome` in the icon search to discover all supported icons:
https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/#search

**Reply by Shivansh-Mittal** [reply]
![image](https://user-images.githubusercontent.com/81565355/171119555-41b202f5-ee17-4997-82ca-ff7098b264af.png)


icon didn't loaded here

**Reply by squidfunk** [reply]
Did you correctly configure the icon integration?
https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/#configuration

**Reply by Shivansh-Mittal** [reply]
in mkdocs.yml
![image](https://user-images.githubusercontent.com/81565355/171141630-1f50a867-d365-4bc0-b186-91713d8d4c6b.png)

in settings.json
![image](https://user-images.githubusercontent.com/81565355/171141678-d65550ed-873d-4a44-a7f6-6cc5c9626886.png)


Yes, I did configure it correctly.

**Reply by Shivansh-Mittal** [reply]
Thank you.
Now it is working.

It was my silly mistake in json file for parenthesis.
As i haven't worked a lot with json.

Now the mkdocs material native icons are working properly.

**Reply by Shivansh-Mittal** [reply]
![image](https://user-images.githubusercontent.com/81565355/171144369-78077c7b-6c17-4a72-8aa4-de8ba2d742ab.png)


icons loading perfectly fine but if i need to use fontawesome then how to setup it?

**Reply by squidfunk** [reply]
As I said, Material bundles all free FontAwesome icons. Otherwise, if you want to set up the FontAwesome icon font, just follow the instructions of FontAwesome. You will likely need to add additional CSS (and maybe JS), see the [customization guide](https://squidfunk.github.io/mkdocs-material/customization/).

**Reply by Shivansh-Mittal** [reply]
Ok. 
Thanks for answering.
